keps/sig-node/4176-cpumanager-spread-cpus-preferred-policy/README.md

```diff
@@ -862,9 +796,13 @@ Major milestones might include:
 
 ## Drawbacks
 
-<!--
-Why should this KEP _not_ be implemented?
--->
+Let's talk about the limitation of current policies.
+
+1. In a cluster with sparse workloads, we try to leverage as much cpu cache as we can. `full-pcpus-only` will always allocate full phsical cores and it introduces cache competition between vcpus.
+
+2. `distribute-cpus-across-num` will evenly distribut CPU across NUMA nodes. In some cases, we want the application to be allocated in single NUMA node if possible, which gives better performance.
+
+Existing solutions can not address all the special needs from high peformance applications, that's why a new option is needed.
```

Comment:
here we need to talk about possible negative effects of _having_ this work implemented. I'd mention code and operational complexity, which are (IMO) offseted by the benefits of having this allocation strategy.

Reply:
I remove this section and update the drawbacks. system wide, there's no drawbacks. workload performance wide, user may have performance regression and it depends on workload pattern